pyproject.toml

```diff
@@ -36,7 +36,7 @@ requires-python = ">=3.8"
 dynamic = ["version"]
 
 [project.optional-dependencies]
-test = ["pytest>=7.0", "mqt.core[qiskit]"]
+test = ["pytest>=7.0", "mqt.core[qiskit]", "pandas>=1.3.0"]
```

Comment:
Please move this to a separate extra `benchmark` or `evaluation` extra and add that extra to the test dependencies. You can literally copy and paste from the way the qiskit extra is handled.